### DISCUSSION TITLE
Error when i try link fields to admin view

### DISCUSSION THREAD

**Discussion by alxzhdanov** [joomengine/Joomla-Component-Builder#938] [Q&A] [answered]
Hello!

Create fields
Create admin view
Press button to link some fiels get error:
Argument 1 passed to VDM\Joomla\Utilities\GetHelper::between() must be of the type string, null given, called in C:\xampp\htdocs\jcb3\libraries\jcb_powers\VDM.Joomla\src\Utilities.php on line 397

And I cant receive confirmaton email from https://git.vdm.dev/ to gmail

**Comment by Llewellynvdm** [comment, marked as answer]
Hi, I fixed the email issue, please open this issue on the git.vdm.dev system... thanks!
